packages/path_provider/path_provider_windows/README.md

```diff
@@ -0,0 +1,32 @@
+# path_provider_windows
+
+The Windows implementation of [`path_provider`][1].
+
+*This is not an official package, and is not ready for use!*
```

Comment:
Remove.

Reply:
Ha! I added that in a vain attempt to satisfy you :) but yeah, I'll remove this.

Reply:
When it's landed, it will be an official package.